src/session.ts

```diff
@@ -280,11 +282,27 @@ export class SessionManager implements Middleware {
                 });
         }
 
+        // If powershell.powerShellDefaultVersion specified, attempt to find the PowerShell exe path
+        // of the version specified by the setting.
+        if ((this.sessionStatus === SessionStatus.NeverStarted) && this.sessionSettings.powerShellDefaultVersion) {
+            const powerShellExePaths = getAvailablePowerShellExes(this.platformDetails, this.sessionSettings);
```

Comment:
I think I'm going crazy but where is `getAvailablePowerShellExes` defined? I see it's in `platform.ts` but not it `session.ts`. How is being called here?

I trust it works, but I'm just curious.

Reply:
It's imported from platform.ts on line 25.